koala/ast/graph.py

```diff
@@ -745,8 +748,11 @@ def gen_graph(self):
             deps = uniqueify(deps)
 
             for dep in deps:
+                if dep in self.named_ranges:
```

Comment:
Could you verify that named range were actually not added to the graph ?

I think maybe this https://github.com/SergeBredin/koala/blob/master/koala/ast/graph.py#L746 does the trick.

I'm saying that because in my basic_evaluation.py example, i have no problem using named ranges and finding the correct result.